controller/lib/core/blower_fsm.h

```diff
@@ -174,7 +183,10 @@ class BlowerFsm {
       }
     }
 
-    return std::visit([&](auto &fsm) { return fsm.desired_state(now); }, fsm_);
+    BlowerSystemState s =
+        std::visit([&](auto &fsm) { return fsm.desired_state(now); }, fsm_);
+    s.is_new_breath = is_new_breath;
+    return s;
```

Comment:
In the current design, each individual FSM only lives for a single breath; see detailed justification for this in review of #458.  So if we put the is_new_breath boolean in the fsm, it would really mean "is first time desired_state() was called").  That's potentially-buggy code we'd have to rewrite in each FSM implementation, plus I don't like that it makes `desired_state` mutate the FSM state.   In my mind, the FSM state should be a function only of the time, desired_state() should be idempotent.

Whereas if we do it this way, the "outer FSM" already needs to have logic to determine if it's a new breath, so we're just using the same logic.

I also don't like returning a partially-filled struct, and I considered splitting it up into two structs, but in the end I thought this was an OK compromise.